src/main/java/org/apache/commons/collections4/bloomfilter/BloomFilter.java

```diff
@@ -60,6 +60,11 @@ public interface BloomFilter extends IndexProducer, BitMapProducer {
      */
     Shape getShape();
 
+    /**
```

Comment:
Why is the clear method being added here? Your PR branch should be rebased on the current master as this change should not be in this PR.